src/nodedef.cpp

```diff
@@ -317,6 +317,18 @@ ContentFeatures::ContentFeatures()
 	reset();
 }
 
+ContentFeatures::~ContentFeatures()
+{
+#ifndef SERVER
+	for (u16 j = 0; j < 6; j++) {
```

Comment:
Please check for `if (tiles)` in case the loading process is interrupted before `fillTileAttribs` was called.